dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -84,6 +95,9 @@ async def get_job_info(self):
                 "end_time": job_table_entry.end_time,
                 "config": config,
             }
+            status = self._get_job_status(metadata)
+            if status is not None:
+                entry["status"] = status
```

Comment:
i think we should always set the status, and just make it None if it doesn't exist. This way we avoid weird key errors on the other side.

Reply:
Ok, sure I'll figure out how to do that. That's what I had originally but jsonschema was not having it.

Reply:
updated